.eslintrc.js

```diff
@@ -1,16 +1,32 @@
 module.exports = {
 	"extends": [
 		"standard",
-		"prettier"
+		"plugin:jest/recommended",
+		"plugin:react/recommended",
+		"prettier",
+		"prettier/react"
 	],
 	"plugins": [
+		"react",
+		"jest",
 		"prettier"
 	],
+    "parser": "babel-eslint",
```

Comment:
Wrong indentation :)

Reply:
Thx :)

PS: this is WIP, maybe you should not spend too much time in PR until finished :)